maas-schemas/schemas/maas-backend/customers/virtual-card-issue/request.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/customers/virtual-card-issue/request.json",
+  "description": "MaaS customer virtual card issue async lambda request",
+  "type": "object",
+  "properties": {
+    "customerId": {
```

Comment:
Should this be `identityId` too?

Reply:
Actually this is for an async lambda, called by another lambda.
As such, it doesn't use APIG/custom authorizer, and thus doesn't have the principal/identityId.